cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/UnwindTest.scala

```diff
@@ -127,7 +127,7 @@ class UnwindTest extends DocumentingTest {
     }
     section("Using `UNWIND` with an expression that is not a list", "unwind-using-unwind-with-an-expression-that-is-not-a-list") {
       p(
-        """Attempting to use `UNWIND` on an expression that does not return a list -- such as `UNWIND 5` -- will cause an error.
+        """Using `UNWIND` on an expression that does not return a list -- such as `UNWIND 5`, will return the same result as using `UNWIND` on a singleton list of that expression, in this case `UNWIND [5]`.
```

Comment:
such as UNWIND 5, will return the same result as using UNWIND on a list that just contains the element 5. So it is effectively equivalent to UNWIND [5]

Reply:
Thanks, I didn't like my own wording. Will update it.